src/hotspot/cpu/x86/c2_stubGenerator_x86_64_string.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024, Intel Corporation. All rights reserved.
+ * Copyright (c) 2024, 2025 Intel Corporation. All rights reserved.
```

Comment:
nit: Incorrect format. One comma is missing. Should be `2025,`

Reply:
Fixed